lib/csrf-token.js

```diff
@@ -0,0 +1,16 @@
+function getCSRFToken() {
+  console.log('Getting csrf token');
+  var url = config.oauthHost + '/users/sign_in/?now=' + Date.now();
```

Comment:
This used `config.host` in the original code but I'm not sure if that was correct. I assume this change is to allow for a different auth host from the Panoptes API host?

Reply:
possibly - that was me copy/pasting. I assumed the config was shared, static data

I assume this file should include `config` lib as well then, might be good to refactor `JSON_HEADERS` into config as well for re-use.

Reply:
Oh yeah, looks like this needs to import the config too. I missed that.

Reply:
Config is static, by the way. It's just a JS dictionary exported from `config.js`.

Reply:
I'll make the changes and push a commit up

Reply:
I know understand the concern re the change of host, my apologies. I'll look into this, it should respect the different host config options.